src/platforms/common/enriching-events/attachments/index.mdx

```diff
@@ -6,6 +6,10 @@ notoc: true
 supported:
   - native
   - javascript
+notSupported:
+  - native.breakpad
```

Comment:
The attachments page is complicated. We do support attachments for all "variants", but only the root (SDK) can add *custom* attachments. 

@PeloWriter is there a way to exclude just a section rather than the entire document?

Reply:
With includes, yes. But not with the frontmatter.